tests/AtelierTomato.Markov.Model.Test/BookObjectOIDTest.cs

```diff
@@ -127,5 +127,23 @@ public void BookParseSentenceNotIntTest()
 			Action act = () => BookObjectOID.Parse("Book:_:Alice in Wonderland:Through the Looking Glass:1:3:Wrong");
 			act.Should().Throw<ArgumentException>().WithMessage("The part of the BookObjectOID corresponding to the sentence was not able to be parsed into an integer value. (Parameter 'OID')");
 		}
+		[Fact]
+		public void IncrementBookOID()
+		{
+			BookObjectOID oid = BookObjectOID.ForParagraph("_", "_", "_", "_", 1);

```

Comment:
would be better if this was distinct values but eh. honestly generation from autofixture would be nice for this